src/mysky/index.ts

```diff
@@ -765,6 +797,10 @@ export class MySky {
     return connection;
   }
 
+  protected async getPreferredPortal(): Promise<string | null> {
```

Comment:
missing docstring

Reply:
Looks like `require-jsdoc` does not require method documentation by default. 🤯

https://eslint.org/docs/rules/require-jsdoc#options

I'll enable stricter checking for this rule.